sdk/eventhub/event-hubs/test/README.md

```diff
@@ -0,0 +1,15 @@
+# Testing
+
+To test this project, make sure to first build it properly by following our [building instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#building). Once the project is correctly built, you will be able to run the tests by following the [testing instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#testing).
+
+The environment variables you will need to properly assign to run the live test of this project are the following:
```

Comment:
```suggestion
The Event Hubs project does not have any recorded tests and so, all the tests require an Azure Event Hubs namespace to be set up beforehand with at lease a single Event Hub instance in it. Set the below environment variables to run the tests
```

Reply:
Use similar format for 
- event-processor-host
- checkpointstore
- service bus

Reply:
on it